index.js

```diff
@@ -44,7 +44,8 @@ export const wrap = async (elements, height, pageNumber) => {
     const futureElements = elements.slice(i + 1)
     const isElementOutside = height <= element.top
     const elementShouldSplit = height < element.top + element.height
-    let elementShouldBreak = element.break || (!element.wrap && elementShouldSplit)
+    const elementFitsFullPageHeight = element.height <= height;
+    let elementShouldBreak = element.break || (!element.wrap && elementShouldSplit && elementFitsFullPageHeight)
```

Comment:
Shoudn't we always break the node if fits the full page height?

```js
let elementShouldBreak = element.break || (!element.wrap && elementShouldSplit) || elementFitsFullPageHeight
```

Reply:
That will make many of the other tests fail, I think. `elementShouldBreak` has precedence over `elementShouldSplit`, so always setting it to true, if the node fits, would break most element to the next page, even if they fit the current page.

My reasoning with the current logic, was to only set `elementShouldBreak` to true, if the element actually fits on the next page - or else fallback to split regardless of `wrap=false`.

Reply:
I guess it's a decision of whether to "split as late as possible" or "split immediately" when a node is to large for page.